src/lib/webpack-config.js

```diff
@@ -260,6 +261,11 @@ export default env => {
 			}
 		}),
 
+		// dev-only plugins
```

Comment:
Ah, right. My hesitation was that it wasn't modifying `plugins` but, ofc, that's what `addPlugins` is doing 😅